src/creator/tyler/Calculator.java

```diff
@@ -62,19 +70,11 @@ else if (stringList.get(i).equals("="))
                 else
                 {
                     System.out.println("Syntax Error");
-                    return;
+                    return Double.NaN;
```

Comment:
You should change this to throw an exception instead, but this should be ok for now.